tests/features/reports.feature

```diff
@@ -345,6 +345,37 @@ Scenario: I can't match JSON page var to str
   wrong_value
   """
 
+@fast @rf22 @login
+Scenario: Fail with wrong email secret
```

Comment:
Could add specific failed message about email address

Reply:
I know what I said, but I've actually realized that since this is in "reports.feature" file, we can assume these are about messages/lines in the report. Sorry about that.